linera-views/benches/stores.rs

```diff
@@ -0,0 +1,547 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    fmt::Debug,
+    time::{Duration, Instant},
+};
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+#[cfg(with_dynamodb)]
+use linera_views::dynamo_db::{create_dynamo_db_test_store, DynamoDbStore};
+#[cfg(with_rocksdb)]
+use linera_views::rocks_db::{create_rocks_db_test_store, RocksDbStore};
+#[cfg(with_scylladb)]
+use linera_views::scylla_db::{create_scylla_db_test_store, ScyllaDbStore};
+use linera_views::{
+    batch::Batch,
+    common::LocalKeyValueStore,
+    memory::{create_memory_store, MemoryStore},
+    test_utils::{get_random_key_values2, prefix_expansion},
+};
+use tokio::runtime::Runtime;
+
+pub async fn performance_contains_key<S: LocalKeyValueStore>(store: S, iterations: u64) -> Duration
```

Comment:
Should we also test not finding the keys? I.e., inserting `num_entries` but then looking for a different set of `num_entries` keys?

Reply:
I added checks with missing keys.
I increased the number of entries to make the test more robust. But I kept the same number of keys over all tests.